### PR TITLE
remove ambiguous variable name 'l'

### PR DESCRIPTION
**Pull request by shenmidelin**
Due to requirement of the new version of `pycodestyle`, the variable `l` is not allowed, so I have removed all of them.

### CODE REVIEW

**Comment by mpmdean** [comment]
Hi @shenmidelin 
I agree it's better to avoid `l` as an iterator variable. 

When we want to refer to angular momentum, however, I think using `l` has a lot of value for readability as physics will all expect `l` to be angular momentum eigenvalue. I would prefer to leave these `l` and add
`ignore = E741`
to the `.flake8` file.

File: edrixs/angular_momentum.py [hunk 1]
```diff
@@ -7,18 +7,18 @@
 from .basis_transform import cb_op, tmat_r2c
 
 
-def get_ladd(l, ispin=False):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:10]: I would revert this chang.

File: edrixs/angular_momentum.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def get_lminus(l, ispin=False):
         return lminus
 
 
-def get_lx(l, ispin=False):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:90]: I would revert this change

File: edrixs/angular_momentum.py [hunk 3]
```diff
@@ -140,25 +140,25 @@ def get_ly(l, ispin=False):
...
 
     if ispin:
-        return (get_ladd(l, True) - get_lminus(l, True)) / 2.0 * -1j
+        return (get_ladd(ll, True) - get_lminus(ll, True)) / 2.0 * -1j
     else:
-        return (get_ladd(l) - get_lminus(l)) / 2.0 * -1j
+        return (get_ladd(ll) - get_lminus(ll)) / 2.0 * -1j
 
 
-def get_lz(l, ispin=False):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:154]: I would revert this change

File: edrixs/angular_momentum.py [hunk 4]
```diff
@@ -185,14 +185,14 @@ def get_lz(l, ispin=False):
         return lz
 
 
-def get_orb_momentum(l, ispin=False):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:188]: I would revert this change

File: edrixs/angular_momentum.py [hunk 5]
```diff
@@ -250,101 +250,101 @@ def get_pauli():
     return sigma
 
 
-def get_sx(l):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:253]: I would revert this change

File: edrixs/angular_momentum.py [hunk 6]
```diff
@@ -250,101 +250,101 @@ def get_pauli():
...
     sx = np.zeros((norbs, norbs), dtype=np.complex128)
     sigma = get_pauli()
-    for i in range(2 * l + 1):
+    for i in range(2 * ll + 1):
         sx[2 * i:2 * i + 2, 2 * i:2 * i + 2] = sigma[0, :, :] / 2.0
 
     return sx
 
 
-def get_sy(l):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:282]: I would revert this change

File: edrixs/angular_momentum.py [hunk 7]
```diff
@@ -250,101 +250,101 @@ def get_pauli():
...
     sy = np.zeros((norbs, norbs), dtype=np.complex128)
     sigma = get_pauli()
-    for i in range(2 * l + 1):
+    for i in range(2 * ll + 1):
         sy[2 * i:2 * i + 2, 2 * i:2 * i + 2] = sigma[1, :, :] / 2.0
 
     return sy
 
 
-def get_sz(l):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:311]: I would revert this change

File: edrixs/angular_momentum.py [hunk 8]
```diff
@@ -250,101 +250,101 @@ def get_pauli():
...
     sz = np.zeros((norbs, norbs), dtype=np.complex128)
     sigma = get_pauli()
-    for i in range(2 * l + 1):
+    for i in range(2 * ll + 1):
         sz[2 * i:2 * i + 2, 2 * i:2 * i + 2] = sigma[2, :, :] / 2.0
 
     return sz
 
 
-def get_spin_momentum(l):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:340]: I would revert this change

File: edrixs/angular_momentum.py [hunk 9]
```diff
@@ -250,101 +250,101 @@ def get_pauli():
...
+def get_spin_momentum(ll):
```
Comment (mpmdean) [inline edrixs/angular_momentum.py:340]: I would revert this change

File: edrixs/angular_momentum.py (hunk 1 above)
Comment (shenmidelin) [reply edrixs/angular_momentum.py:10]: I have tried to add "ignore = E741" in .flake8 file, however, it gives more other warnings. I do not want to spend time to fix those new warnings, so let's do not use `l` in all the places to make life simple.

**Comment by mrakitin** [comment]
Is docker building process working fine now?

**Comment by shenmidelin** [comment]
Travis CI building is sucessful now.

**Comment by shenmidelin** [comment]
Travis CI building is sucessful now.

**Comment by mpmdean** [comment]
But as @shenmidelin wrote to me separately. It seems like dockerfile_interactive is not.

Weirdly I cannot reproduce the error on my mac?

My proposed solution would to copy these very competent folks
https://github.com/jupyter/docker-stacks/blob/master/scipy-notebook/Dockerfile
which I'm already partially doing and to pin 
jupyterlab=2.1.3

It looks like it's currently using jupyterlab=2.1.2

I'll do this unless you have a different idea?

**Comment by mpmdean** [comment]
See #49

**Comment by mpmdean** [comment]
It looks like this is solved.  

For future reference, edrixs_interactive &  edrixs autobuild on dockerhub whereas the base image needs to be manually triggered.
